contracts/Tokenomics.sol

```diff
@@ -246,6 +247,13 @@ struct StakingPoint {
     uint8 stakingFraction;
 }
 
+// Struct for service staking epoch info
+struct DonationPoint {
+    uint96 totalUnitTopUps;
+    uint96 totalDonationPower;
+    bool moreDonationPower;
+}
```

Comment:
Donation point, clearly the comment is wrong. It's a bit formatted in a next PR.